helix-core/src/test/java/org/apache/helix/integration/paticipant/TestInstanceHistory.java

```diff
@@ -44,6 +46,13 @@
     List<String> list = history.getRecord().getListField("HISTORY");
     Assert.assertEquals(list.size(), 1);
 
+    Assert.assertTrue(list.get(0).contains("SESSION=" + _participants[0].getSessionId()));
+    Assert.assertTrue(list.get(0).contains("VERSION=" + _participants[0].getVersion()));
+    Assert
+        .assertTrue(list.get(0).contains("HOST_NAME=" + InetAddress.getLocalHost().getHostName()));
```

Comment:
Nit, this `InetAddress.getLocalHost().getHostName()` doesn't seem to converge to the code. If there is a case this line throws `UnknownHostException`, it makes the test flaky. Do you think we should also this the test converge to the main code?

Reply:
Agreed.